scripts/haas_network.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=HaaS Network Server 
+After=network.target
+
+[Service]
+User=haas_user
+Group=haas_user
+WorkingDirectory=/var/lib/haas/
+ExecStart=/usr/bin/haas serve_networks
```

Comment:
``KillMode=process`` will kill only the main process. 
Since we are running only one process I thought to state it explicitly on how to kill it.

In general I want to be as much explicit as possible and not depend on defaults. 
That gives more control and consistency. 
Same reason for using ``Type=simple`` 

ExecReload was set to replicate the same behavior as what we would do manually to restart a process. But in retrospect (and did some reading) I guess it is a not a good idea to use  ``kill -HUP`` command as it is asynchronous.